### DISCUSSION TITLE
Minecraft is getting better in every update but i will like to suggest you to add some features and maps like left hane torch, dynamic lightening also map like sky block oneblock and hardcore. I kindly request you to add this feature in next update. This is my big request. So plz take my request serously. Thanks for this game. Hope to see above features in next update.

### DISCUSSION THREAD

**Discussion by SagarDulal09** [Mojang/minecraft-editor#116] [Feedback]
Minecraft is getting better in every update but i will like to suggest you to add some features and maps like left hane torch, dynamic lightening also map like sky block oneblock and hardcore. I kindly request you to add this feature in next update. This is my big request. So plz take my request serously. Thanks for this game. Hope to see above features in next update.

**Comment by ihategravel2** [comment]
Sir, This Is A Wendy's. (Also known as Editor Mode)

**Comment by minecraftchloe** [comment]
Thanks for your suggestion! This is a repo for the Bedrock Editor, though. I'd encourage you to post your idea to the main game's feedback site: https://feedback.minecraft.net/hc/en-us. Closing this.
